windeploy_helper.sh

```diff
@@ -33,10 +33,19 @@ elif [[ "$platform" == "mingw32" ]]; then
 	PLATFORM_FILES="libgcc_s_dw2-1.dll" 
 fi
 
-for f in $FILES; do cp $MSYSTEM_PREFIX/bin/$f $TARGET; done
+for f in $FILES; do cp $MSYSTEM_PREFIX/bin/$f $TARGET; if [ "$?" -ne "0" ]; then
```

Comment:
If you still want to add error checking after fixing the array issue, this would be a bit cleaner:
 ```bash
for f in $FILES; do cp $MSYSTEM_PREFIX/bin/$f $TARGET || exit 1; done
```